spec/openapi.yaml

```diff
@@ -2236,6 +2239,9 @@ components:
           description: Vat percentage
           type: string
           deprecated: true
+        taxable:
+          description: Whether tax is charged when the product is sold
```

Comment:
```suggestion
          description: Whether tax is charged when the article is sold
```

Reply:
Directly copied from Bas his example

Reply:
not consistent with all other descriptions 😕